library/src/main/kotlin/com/livefront/gsonkotlinadapter/KoltinReflectiveTypeAdapterFactory.kt

```diff
@@ -40,61 +40,56 @@ class KotlinReflectiveTypeAdapterFactory private constructor(
         if (!rawType.isAnnotationPresent(KOTLIN_METADATA)) return null
         val kotlinRawType: KClass<T> = type.toKClass()
         require(!kotlinRawType.isInner) { "Cannot serialize inner class ${rawType.name}" }
-        kotlinRawType.primaryConstructor ?: return null
```

Comment:
Just realized the file name begins with `Koltin` 😆

Reply:
lol